base/iobuffer.jl

```diff
@@ -55,10 +55,21 @@ It may take optional keyword arguments:
 - `read`, `write`, `append`: restricts operations to the buffer; see `open` for details.
 - `truncate`: truncates the buffer size to zero length.
 - `maxsize`: specifies a size beyond which the buffer may not be grown.
-- `sizehint`: suggests a capacity of the buffer (`data` must implement `sizehint!(data, size)`).
+- `sizehint`: suggests a capacity of the buffer (`data` must implement `sizehint!(data,
+  size)`).
```

Comment:
We have no strict line length limit
```suggestion
- `sizehint`: suggests a capacity of the buffer (`data` must implement `sizehint!(data, size)`).
```